app/src/main/java/com/mho/training/sources/PersonDataSource.kt

```diff
@@ -17,24 +17,24 @@ import java.io.IOException
 
 class PersonDataSource(private val resources: Resources) : RemotePersonDataSource {
```

Comment:
You could specify the name of the data source; for example:

PersonServerDataSource

Reply:
Got it!